metroae

```diff
@@ -677,42 +677,85 @@ function destroy {
 
 function pull {
     debug ${FUNCNAME[0]}
-    if [[ ! -z $1 ]]; then
-        MAX_CONTAINER_VERSION=$1
-    fi
-    debug "${FUNCNAME[0]}: MAX_CONTAINER_VERSION: $MAX_CONTAINER_VERSION"
+    if [[ ! -z $1 ]] && [[ $1 == "--repo" ]]; then
+        REPO=$2
+        debug "${FUNCNAME[0]}: REPO: $REPO"
+        
+        set -e
 
-    set -e
+        echo ""
+        write_to_screen_and_script_log ">>> Pulling the MetroAE container image from Docker hub"
+        echo ""
 
-    echo ""
-    write_to_screen_and_script_log ">>> Pulling the MetroAE container image from Docker hub"
-    echo ""
-    if [[ -f $SCRIPT_LOG_FILE ]]; then
-        if [ -w "$SCRIPT_LOG_FILE" ]; then
-            debug "${FUNCNAME[0]}: Script log is writable"
-            sudo docker pull $METRO_AE_IMAGE:$MAX_CONTAINER_VERSION | tee -a $SCRIPT_LOG_FILE
+        if [[ -f $SCRIPT_LOG_FILE ]]; then
+            if [ -w "$SCRIPT_LOG_FILE" ]; then
+                debug "${FUNCNAME[0]}: Script log is writable"
+                sudo docker pull $REPO | tee -a $SCRIPT_LOG_FILE
```

Comment:
what is the format of the command line argument? if we do: `metroae pull --repo some.repo.com`  then we need to append `metroae` (and a trailing slash if not provided by the user already). So something like `sudo docker pull ${REPO}metroae` (assuming that REPO contains the trailing slash)